qa/L0_infer_reshape/infer_reshape_test.py

```diff
@@ -308,7 +308,7 @@ def test_ii3(self):
         self._full_reshape(np.int32, input_shapes=([1, 4, 1], [8], [2, 2, 3]))
 
     def test_oo1(self):
-        self._full_reshape(np.object, input_shapes=([1],), no_batch=False)
+        self._full_reshape(np.object_, input_shapes=([1],), no_batch=False)
```

Comment:
Why do we use "object_" here but elsewhere use "object"?

Reply:
I tried to use `np.object_` everywhere. `np.object_` and `object` are interchangeable.